ddl/ddl_tiflash_api.go

```diff
@@ -287,6 +293,49 @@ func GetTiFlashReplicaInfo(tblInfo *model.TableInfo, tableList *[]TiFlashReplica
 	}
 }
 
+// GetTiFlashReplicaMapInfo parses tableID + model.TableInfo into map[int64]TiFlashReplicaStatus
+func GetTiFlashReplicaMapInfo(tblInfo *model.TableInfo, tableMap *map[int64]TiFlashReplicaStatus) {
```

Comment:
Actually `tableMap` can be `map[int64]TiFlashReplicaStatus`.

See https://go.dev/play/p/pb3VZdLLJOo

```go
package main

import "fmt"

func foo(v map[int]int) {
	v[1] = 2
}

func main() {
	x := map[int]int{}
	foo(x)
	fmt.Println(x[1])
}
```

BTW, maybe it's better to indicate that this function write result into something. How about naming it as:

```go
func writeTiFlashReplicaInfo(tblInfo *model.TableInfo, intoMap map[int64]TiFlashReplicaStatus)
```

Reply:
just keep the style with `GetTiFlashReplicaInfo`